Pod/Classes/MenuView.swift

```diff
@@ -8,6 +8,8 @@
 
 import UIKit
 
+internal let MenuViewItemsMarginRigth: CGFloat = 20
```

Comment:
I understand that it makes easier, but I actually prefer not to have static value. Anyway I'll wait for [this answer](https://github.com/kitasuke/PagingMenuController/issues/291#issuecomment-274369894) to understand what the case is at first.